openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -0,0 +1,19 @@
+"""
+API Serializers for taxonomies
+"""
+
+from rest_framework import serializers
+
+
+class TaxonomySerializer(serializers.Serializer):
+    """ Serializer for taxonomy """
+    id = serializers.IntegerField()
+    name = serializers.CharField()
+    description = serializers.CharField()
+    enabled = serializers.BooleanField()
+    required = serializers.BooleanField()
+    allow_multiple = serializers.BooleanField()
+    allow_free_text = serializers.BooleanField()
+    system_defined = serializers.BooleanField()
+    visible_to_authors = serializers.BooleanField()
+    _object_tag_class = serializers.CharField()
```

Comment:
No need to serialize this field -- it's for backend use only, and so isn't needed by users of the REST API.